src/app.js

```diff
@@ -1,9 +1,12 @@
 const express = require('express');
 const router = require('./controllers/router.js');
 const hbs = require('express-handlebars');
+const languages = require('./lang.js');
 
 const app = express();
 
+app.locals.text = languages['english'];
```

Comment:
we're not always going to want this to be english.

Could do:

```
app.locals.textEn = languages['english']
app.locals.textAr = languages['arabic']
```

Actually, why not `languages.english`?